### PR TITLE
Apply band mask in addition to RFI mask to skip channels

### PR DESCRIPTION
**Pull request by bmerry**
This makes little difference to performance at the moment, but makes the
channel status in the report more representative.

### CODE REVIEW

**Comment by bmerry** [comment]
I'm aware that this duplicates a fair amount of code also found in katsdpcontroller. I'm going to leave that as a problem for katdal to solve.

File: katsdpimager/loader_katdal.py [hunk 1]
```diff
@@ -187,16 +188,30 @@ def __init__(self, filename, options):
             corrections = 'none'
         _logger.info('Calibration corrections applied: %s', corrections)
 
-        # Determine RFI mask, if any (TODO: eventually katdal should do some of this)
+        # Determine RFI and band mask, if any. Using the band mask isn't
```
Comment (sratcliffe) [inline katsdpimager/loader_katdal.py:191]: On line 115 (parser for rfi-mask argument) we should probably mention that this now applies both the RFI and Band masks. I had first thought of splitting these into two arguments, but that seems overkill.

Comment (bmerry) [reply katsdpimager/loader_katdal.py:191]: Good point. Done, and also added to the user manual.

**Review by sratcliffe — APPROVED** [review]
Looks good

**Pull request merged**